pkg/controller/cassandracluster/reconcile.go

```diff
@@ -640,6 +641,10 @@ func FlipCassandraClusterUpdateSeedListStatus(cc *api.CassandraCluster, status *
 				dcRackName := cc.GetDCRackName(dcName, rackName)
 				dcRackStatus := status.CassandraRackStatus[dcRackName]
 
+				if dcRackStatus == nil {
```

Comment:
which one is actually failing ? this one or the latter ? Do we know why it happens ? It seems we are solving it down front but something happens up front ...

Reply:
Both actually.. When I have fixed the 1st dcRackStatus assignment, which cause the operator to crash as logs said, the operator still crashed because of the second. It's like the value of cc & status in parameters change over the execution of the function.

Since the problem of crash is most likely caused by a read/write concurrency on CC status attribute and since parameters are pointers, I think this is the cause.

I understand that this patch not really solves the problem, but it solves the consequences of it. And because of the problem is a concurrency one, it seems fine (to me) to just wait for 5 seconds to correctly finish the 'write' operation and read again 5s later since it's avoiding the issue.